src/Cli/Microsoft.DotNet.Configurer/Microsoft.DotNet.Configurer.csproj

```diff
@@ -9,6 +9,7 @@
     <RepositoryType>git</RepositoryType>
     <DefineConstants Condition="'$(IncludeAspNetCoreRuntime)' == 'false'">$(DefineConstants);EXCLUDE_ASPNETCORE</DefineConstants>
     <IsPackable>true</IsPackable>
+    <Nullable>enable</Nullable>
```

Comment:
Should this be in a `directory.build.props` perhaps?

Reply:
After some discussion, it seems like the forward strategy will be:
1. Have `#nullable enable` at the top of **every shared source file** (won't be in this PR, but as a likely follow-up)
2. Progressively convert projects by adding `<Nullable>enable</Nullable>` to them
3. When all projects are converted, add `<Nullable>enable</Nullable>` to the main Directory.Build.props, and remove the `#nullable enable` from the shared source files.